boilerplate/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright YEAR The Kubernetes Authors.
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Pretty sure this is not the right place for this change but since https://github.com/kubernetes/gengo/pull/91 hasn't merged yet, this is necessary to check that the behavior of the example generators isn't affected.